rdt/transformers/base.py

```diff
@@ -74,16 +74,16 @@ def get_supported_sdtypes(cls):
         """
         return cls.SUPPORTED_SDTYPES or [cls.INPUT_SDTYPE]
 
-    def _add_prefix(self, sdtype_or_transformer):
+    def _get_output_to_property(self, property_):
```

Comment:
is the underscore because property is a shadowed name?

Reply:
yep